sdk/core/azure-core/azure/core/pipeline/transport/base.py

```diff
@@ -43,21 +43,22 @@
 # If one day we reach the point where "requests" can be skip totally,
 # might provide our own implementation
 from requests.structures import CaseInsensitiveDict
-from azure.core.pipeline import ABC, AbstractContextManager
+from azure.core.pipeline import ABC, AbstractContextManager, PipelineRequest, PipelineResponse
```

Comment:
Maybe we should use PipelineRequestType instead of import?
Can this change pass PyLint?

Reply:
yes, it does.